tableauserverclient/models/datasource_item.py

```diff
@@ -104,22 +130,30 @@ def from_response(cls, resp):
         all_datasource_xml = parsed_response.findall('.//t:datasource', namespaces=NAMESPACE)
 
         for datasource_xml in all_datasource_xml:
-            (id, name, datasource_type, content_url, created_at, updated_at,
-             tags, project_id, project_name, owner_id) = cls._parse_element(datasource_xml)
+            (id_, name, datasource_type, content_url, created_at, updated_at,
+             tags, project_id, project_name, owner_id,
+             certified, certification_note) = cls._parse_element(datasource_xml)
             datasource_item = cls(project_id)
-            datasource_item._set_values(id, name, datasource_type, content_url, created_at, updated_at,
-                                        tags, None, project_name, owner_id)
+            datasource_item._set_values(id_, name, datasource_type, content_url, created_at, updated_at,
+                                        tags, None, project_name, owner_id, certified, certification_note)
             all_datasource_items.append(datasource_item)
         return all_datasource_items
 
     @staticmethod
     def _parse_element(datasource_xml):
-        id = datasource_xml.get('id', None)
+        blorp = datasource_xml
+        id_ = datasource_xml.get('id', None)
         name = datasource_xml.get('name', None)
         datasource_type = datasource_xml.get('type', None)
         content_url = datasource_xml.get('contentUrl', None)
         created_at = parse_datetime(datasource_xml.get('createdAt', None))
         updated_at = parse_datetime(datasource_xml.get('updatedAt', None))
+        certification_note = datasource_xml.get('certificationNote', None)
```

Comment:
Technically yes. The UI may choose to not display them (yet) but it's not expressly forbidden. We may also add UI support for this anywho.